CogNative/backend/modules/STT.py

```diff
@@ -1,6 +1,6 @@
 import speech_recognition as sr
 
-from modules import languages
+from .languages import available_languages
```

Comment:
Is this change okay? With this change, the importing doesn't working when running the script independently from the ```backend/modules``` directory, but it works when importing this module from ```main.py``` (or any other parent directory).

The previous ```from modules import languages``` worked when running the script from ```backend/modules```, but did not work when accessing it from a parent directory.